Gemfile

```diff
@@ -5,7 +5,7 @@ ruby '~> 2.6.10'
 
 gem 'puma'
 gem 'rack-rewrite'
-gem 'rails', '~> 6.0.3.3'
+gem 'rails', '~> 6.0.4.8'
```

Comment:
Let's also update documentation in our wiki to this version of Rails!

Reply:
Sure @julianguyen Once you'll merge this PR, ill immediately update the version number [here](https://github.com/ifmeorg/ifme/wiki/Installation). Is there any other place or other wiki where version number of Rails should be updated?